drip/admin.py

```diff
@@ -34,10 +37,16 @@ class DripAdmin(admin.ModelAdmin):
     form = DripForm
     users_fields: Union[str, List[str]] = []
 
-    def av(self, view):
+    def av(self, view: Callable) -> Callable:
         return self.admin_site.admin_view(view)
 
-    def timeline(self, request, drip_id, into_past, into_future):
+    def timeline(
+        self,
+        request: WSGIRequest,
```

Comment:
can you use  HttpRequest instead of WSGIRequest or they are different?

Reply:
This two classes are "similar" but checking the types here, this functions are getting a `WSGIRequest`.